### PR TITLE
added jax numpy namespace function add

### LINKED ISSUE #6317: sum

### PR DESCRIPTION
**Pull request by Nunyapa**
Close #6317

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py [hunk 1]
```diff
@@ -1594,3 +1594,40 @@ def test_jax_lax_cumprod(
...
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="jax",
+        fn_tree="numpy.sum",
+        a=x,
+        axis=axis,
+        dtype=input_dtype,
+        out=None,
+        keepdims=keepdims,
```
Comment (guillesanbri) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:1632]: same as above, the test should reflect all the arguments in the original backend function

**Comment by guillesanbri** [comment]
Hi! Thank you very much for your contribution to Ivy! I have added a few comments regarding some missing arguments that should be addressed. If you have any question I can help with don't hesitate to ask 🙂​​

**Comment by Nunyapa** [comment]
Hello [guillesanbri](https://github.com/guillesanbri)! Thank you for your review:) I'll be back with questions or with a PR:)

**Comment by guillesanbri** [comment]
sounds great! just to clarify, you can just make more commits to your fork and they will be reflected in this PR, no need to do another one :slightly_smiling_face:

**Comment by Nunyapa** [comment]
Yeah, looks like I did something wrong here. Sorry for that :( I still want to continue to work on the task.

**Comment by guillesanbri** [comment]
I think you have force-pushed some ivy commit to your branch and therefore you have overrided your initial changes :sweat_smile: , I cloned your repo and have a local copy of your commits, in case it saves you some time

```
@to_ivy_arrays_and_back
def sum(a, axis=None, dtype=None, out=None, keepdims=False, initial=None, where=None, promote_integers=True):
    return ivy.sum(a, axis=axis, dtype=dtype, keepdims=keepdims, out=out)
```

```
@handle_cmd_line_args
@given(
    dtype_and_x=helpers.dtype_values_axis(
        available_dtypes=helpers.get_dtypes("numeric")
    ),
    num_positional_args=helpers.num_positional_args(
        fn_name="ivy.functional.frontends.jax.numpy.sum"
    ),
    keepdims=st.booleans(),
)
def test_jax_numpy_sum(
    dtype_and_x,
    as_variable,
    with_out,
    num_positional_args,
    native_array,
    keepdims,
    fw,
):
    input_dtype, x, axis = dtype_and_x
    helpers.test_frontend_function(
        input_dtypes=input_dtype,
        as_variable_flags=as_variable,
        with_out=with_out,
        num_positional_args=num_positional_args,
        native_array_flags=native_array,
        fw=fw,
        frontend="jax",
        fn_tree="numpy.sum",
        a=x,
        axis=axis,
        dtype=input_dtype,
        out=None,
        keepdims=keepdims,
    )
```

**Comment by guillesanbri** [comment]
If you commit to your branch now either me or you can maybe reopen the PR, not sure but worth trying imo!

**Comment by Nunyapa** [comment]
Ok, thank you so much:)

**Comment by guillesanbri** [comment]
No problem :smiley:

**Pull request closed without merging**